pkgs/development/libraries/tiledb/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+lib
+,stdenv
+,fetchgit
+,setuptools_scm
+,doxygen
+,cmake
+,zlib
+,lz4
+,bzip2
+,zstd
+,spdlog_0
+,tbb
+,openssl
+,boost
+,libpqxx
+,catch
+,python
+}:
+
+stdenv.mkDerivation {
+    name = "tiledb";
+    version = "1.5.1";
+
+  src = fetchgit {
+    url = "https://github.com/TileDB-Inc/TileDB";
+    rev = "7236bb29f2aa1ac4de489f0791e9265cb257184d";
+    sha256 = "sha256:0ky0dcv1w1jn1cjn3819aq9xyd2wg80aagf2flxmd916flgr9zjl";
+  };
+
+  makeTarget = "tiledb";
+
+  outputs = ["out"];
+
+  buildInputs = [doxygen cmake zlib lz4 bzip2 zstd spdlog_0 tbb openssl boost libpqxx catch python];
```

Comment:
The `doxygen` should be `nativeBuildInputs` because you need to "run" it during build time.

Reply:
Also doxygen is only used if you also build the docs. Look into the tiledb build instructions for building docs, and use multiple output targets.

Reply:
https://nixos.org/nixpkgs/manual/#chap-multiple-output